webapi/cvmfsmon_api.py

```diff
@@ -47,6 +47,7 @@
 aliases = {}
 excludes = []
 disables = []
+updated_slowrepo = []
```

Comment:
This is a list so let's give it a plural name, that is, `updated_slowrepos`

Reply:
Added comments to the config file and changed the updated-slowrepo name: https://github.com/vzokay/cvmfs-servermon/commit/47bf040ec916ab670046618456dfcaab09ffd53a